### ISSUE TITLE
[Feature] Scalatest is not able to create unite "index.html" report for different modules/subprojects

### ISSUE THREAD

**Issue by Shadowrazej** [scalatest/scalatest#2231]
Issue is described here - https://stackoverflow.com/questions/23961332/can-scalatest-generate-a-single-html-report-for-all-subprojects

**Comment by cheeseng** [comment]
@Shadowrazej Yes unfortunately currently this is not supported, I am not sure what's the best way to support it, we'll need something to aggregate the results of sub-project, I am not sure if we should do it from scalatest core or a sbt-plugin.
